Tasks/SonarQubePostTest/PRCA/ReportProcessor-Module.psm1

```diff
@@ -28,12 +28,19 @@ function FetchAnnotatedNewIssues
     CreateComponentKeyToPathMap $json
 
     # '@' makes sure the result set is returned as an array
-    $newIssues = @($json.issues | Where {$_.isNew -eq $true})
+    $newIssues = @($json.issues | Where { $_.isNew -eq $true })
     Write-Host "SonarQube found $($json.issues.Count) issues out of which $($newIssues.Count) are new"
+    
+    $newFileLevelIssues = @($newIssues | Where {(IsFileLevelIssue $_)})
+    $difference = $newIssues.Count - $newFileLevelIssues.Count
+    if ($difference -gt 0)
+    {
+        Write-Host "$difference issue(s) are not at the file level issues and will not be posted to the code review"
```

Comment:
Sentence isn't correct: _at the file level issues_.
Either _are not at file-level_, or _are not file-level issues_.

It might clearer to change the message to something like "X issue(s) do not relate to a specific file..."